spec/support/pages/edit_user.rb

```diff
@@ -7,5 +7,6 @@ class EditUser < SitePrism::Page
       text: 'Sorry, you don\'t seem to have the resource_manager permission for this app.'
     )
     element :flash_of_success, '.alert-success'
+    element :delete_button, '.t-delete'
```

Comment:
I think we can drop the name of the thing. Gives us scope to change the thing if we need to and usually the naming is enough context.